src/main/java/com/iota/iri/service/restserver/resteasy/ApiPath.java

```diff
@@ -0,0 +1,60 @@
+package com.iota.iri.service.restserver.resteasy;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Request;
+import javax.ws.rs.core.Response;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.iota.iri.service.dto.AbstractResponse;
+import com.iota.iri.service.restserver.ApiCall;
+import com.iota.iri.service.restserver.ApiProcessor;
+
+@Path("")
+public class ApiPath extends ApiCall {
+    
+    @Context
+    HttpHeaders requestHeaders;
+    
+    @Context
+    HttpServletRequest httpRequest;
+
+    @Context
+    Request request;
+    
+    public ApiPath() {
+        
+    }
+    
+    @GET
+    @Path("ping")
+    public String ping() {
+        return "pong";
+    }
+    
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response apiCall(String request) throws JsonParseException, JsonMappingException, IOException {
+        AbstractResponse response = pre(request);
+        if (response == null) {
+           response = process();
+        }
+
+        return Response.ok(post(response)).build();
+    }
+
+    private AbstractResponse process() {
+        return AbstractResponse.createEmptyResponse();
+    }
+}
```

Comment:
I could make it work, but it would change all API paths and calling method. So i left it at this for now.

I left it for us to decide what to do with them. Ill stash them in a local branch